templates/angular/igx-ts/generate/template.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "$(templateName)",
+	"name": "$(templateName)",
+	"description": "$(templateName) template",
+	"framework": "$(templateFramework)",
+	"projectType": "$(templateType)",
+	"components": "<add array of necessary components here>",
+	"controlGroup": "<add your mandatory control group name here>",
+	"listInCustomTemplates": true,
+	"dependencies": "<add array of necessary components' dependencies here>"
```

Comment:
This property is array so the generated value should be an empty array "dependencies": []
Unfortunately we can't place comment so it will be empty.